metadata.json

```diff
@@ -28,22 +28,61 @@
         "14.04",
         "16.04",
         "18.04",
-        "18.10"
+        "18.10",
+        "20.04"
       ]
     },
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "5",
         "6",
-        "7"
+        "7",
+        "8",
+        "9"
+      ]
+    },
+    {
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "6",
```

Comment:
Do we need to list the EoL version 6?

Reply:
@bastelfreak We don't but I didn't want to cause another round of updates by major version bumping this thing.

Reply:
Sorry, in my head that was the RHEL entry. Yeah, we don't have to add it. Parity with RHEL seemed nice though.